rack-throttle.gemspec

```diff
@@ -29,10 +29,11 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version      = '>= 1.8.2'
   gem.requirements               = []
+  gem.add_runtime_dependency     'bundler',   '>= 1.0.0'
```

Comment:
Finally you added bundler as runtime dependency, why this is needed?

Reply:
This was supposed to be a development_dependency, but mistakenly got added as a runtime_dependency.

Reply:
It is still a runtime_dependency, are you going to fix this?